testing/test_faulthandler.py

```diff
@@ -98,6 +98,33 @@ def test_cancel_timeout_on_hook(monkeypatch, hook_name):
 
     # call our hook explicitly, we can trust that pytest will call the hook
     # for us at the appropriate moment
-    hook_func = getattr(plugin_module, hook_name)
-    hook_func()
+    hook_func = getattr(FaultHandlerHooks, hook_name)
+    hook_func(self=None)
     assert called == [1]
+
+
+def test_already_initialized(testdir):
+    """AttributerError when faulthandler is initialized earlier than pytest (#6575)"""
```

Comment:
```suggestion
    """Test for faulthandler being initialized earlier than pytest (#6575)"""
```
Seen a typo, but think it makes sense to rephrase.